scripts/analyze_failed_jobs/action_plugins/parsejobs.py

```diff
@@ -0,0 +1,40 @@
+import glob, pickle, buildbot, re, sys, cStringIO as StringIO
```

Comment:
Well that's an Ansible module, its not meant to be used directly, its called by Ansible. Maybe I should add a README.md to explain how to run the playbook with Ansible?